src/VolumeDrawable.ts

```diff
@@ -62,44 +52,19 @@ export default class VolumeDrawable {
 
   private volumeRendering: VolumeRenderImpl;
 
-  private bounds: Bounds;
-  private scale: Vector3;
-  private currentScale: Vector3;
   private renderUpdateListener?: (iteration: number) => void;
-  private density: number;
-  private brightness: number;
 
   constructor(volume: Volume, options: VolumeDisplayOptions) {
     this.PT = !!options.renderMode;
 
     // THE VOLUME DATA
     this.volume = volume;
+    this.settings = defaultVolumeRenderSettings();
+    VolumeRenderSettingUtils.resizeWithVolume(this.settings, volume);
 
     this.onChannelDataReadyCallback = undefined;
 
-    this.translation = new Vector3(0, 0, 0);
-    this.rotation = new Euler();
-    this.flipX = 1;
-    this.flipY = 1;
-    this.flipZ = 1;
-
-    // TODO: Replace with enum
-    this.viewMode = "3D";
-
-    this.maskChannelIndex = -1;
-
-    this.maskAlpha = 1.0;
-
-    this.gammaMin = 0.0;
-    this.gammaLevel = 1.0;
-    this.gammaMax = 1.0;
-
-    // TODO verify that these are reasonable
-    this.density = 0;
-    this.brightness = 0;
-
-    this.showBoundingBox = false;
-    this.boundingBoxColor = [1.0, 1.0, 0.0];
+    this.viewMode = Axis.NONE;
```

Comment:
none is the same as 3D?

Reply:
I don't mind using "NONE" but maybe a comment here to say that NONE means full 3d volumetric view.  Maybe even an additional enum value that aliases to NONE.  `_3D` is ok, or maybe `VOLUMETRIC` or `XYZ`.

Reply:
At least a comment would be good, the rest is optional IMO.